docs/src/main/paradox/bootstrap/demos/kubernetes-api.md

```diff
@@ -1,20 +1,26 @@
-### Kubernetes API 
+# Kubernetes API 
 
-This demo can be found under `bootstrap-demo/kubernetes-api`. 
+An example project that can be deployed to kubernetes via `minikube` is in `bootstrap-demo/kubernetes-api`.
 
-In Kubernetes, Akka Cluster can be deployed as a single [Headless Service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services).
+This demo shows how to form an Akka Cluster in Kubernetes using the `kubernetes-api` discovery mechanism. The `kubernetes-api`
+mechanism queries the Kubernetes API server to find pods with a given label. A Kubernetes service isn't required 
+for the cluster bootstrap but may be used for external access to the application. It can be be found under `bootstrap-demo/kubernetes-api`. 
 
-An example application using docker and prepared to be deployed to kubernetes is provided in Akka Management's github repository 
-as sub-project [bootstrap-joining-demo](https://github.com/akka/akka-management/tree/master/bootstrap-joining-demo/kubernetes-api).
+The following Kubernetes resources are created:
 
-Rather than configuring the Dockerfile directly, we used the [sbt-native-packager](http://sbt-native-packager.readthedocs.io/en/stable/) 
-to package the application as docker container. See the `build.sbt` file for more details, and the `kubernetes/akka-cluster.yml` 
-file for the service configuration, which is:
+* Deployment: For creating the Akka pods 
+* Role and RoleBinding to give the pods access to the API server
 
 @@snip [akka-cluster.yml]($management$/bootstrap-demo/kubernetes-api/kubernetes/akka-cluster.yml) 
 
-You run the example using [minikube](https://github.com/kubernetes/minikube) (or a real kubernetes system),
-which you can do by typing:
+The following configuration is required:
+
+* Set `akka.discovery.method` to `kubernetes-api`
+* Set `akka.discovery.kubernetes-api.pod-label-selector` to a label selector that will match the Akka pods
+
+@@snip [akka-cluster.yml]($management$/bootstrap-demo/kubernetes-api/src/main/resources/application.conf) { #discovery-config } 
+
+You can run the example using [minikube](https://github.com/kubernetes/minikube) (or a real Kubernetes system) by:
```

Comment:
The instructions below expect minikube though. Perhaps split this up in "set up K8s, for example like this with Minikube or another K8s system" and then pick up at step 4 for 'general' instructions again?

Reply:
:+1: